components/association-section/index.tsx

```diff
@@ -36,33 +31,69 @@ const AssociationNotFound: React.FC<{
         ['N°RNA', formatIntFr(uniteLegale.association.idAssociation || '')],
       ]}
     />
-  </Section>
+  </>
 );
 
-const AssociationSection: React.FC<{
+export default function AssociationSection({
+  uniteLegale,
+}: {
   uniteLegale: IAssociation;
-}> = ({ uniteLegale }) => {
+}) {
   const {
     association: { idAssociation = '', data },
   } = uniteLegale;
-
   if (!data) {
-    return <AssociationNotFound uniteLegale={uniteLegale} />;
-  }
-
-  if (isAPINotResponding(data)) {
-    if (data.errorType === 404) {
-      return <AssociationNotFound uniteLegale={uniteLegale} />;
-    }
-
-    return (
-      <AdministrationNotResponding
-        administration={data.administration}
-        errorType={data.errorType}
-      />
-    );
+    // This case should not happen : if AssociationSection is rendered, it means
+    // that the uniteLegale is an association, and so data is not null (though it
+    // can be of type IAPINotRespondingError).
+    return null;
```

Comment:
Not sure to understand why `<AssociationNotFound uniteLegale={uniteLegale} />;`became `null` ?